src/components/selectionBoxLayer.ts

```diff
@@ -183,9 +225,21 @@ export module Components {
      *
      * Returns an undefined array if the edges are not backed by a scale.
```

Comment:
Since this is a public method, we need more thorough documentation. What happens in general, how to use this method, what are the edge cases. We just have edge cases above